source/Octopus.Tentacle.Client/Execution/RpcCallExecutorFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Octopus.Tentacle.Client.Retries;
+using Octopus.Tentacle.Contracts.Observability;
+using Polly.Timeout;
+
+namespace Octopus.Tentacle.Client.Execution
+{
+    internal static class RpcCallExecutorFactory
```

Comment:
It doesn't look like this is used in Tentacle code, just test code. Was that intentional?

Reply:
No, I just forgot to use it (highlighted in your previous comment 😁)